src/recipes/editor-plugin/inspector-plugins.md

```diff
@@ -0,0 +1,342 @@
+<!--
+  ~ Copyright (c) godot-rust; Bromeon and contributors.
+  ~ This Source Code Form is subject to the terms of the Mozilla Public
+  ~ License, v. 2.0. If a copy of the MPL was not distributed with this
+  ~ file, You can obtain one at https://mozilla.org/MPL/2.0/.
+-->
+
+# Inspector plugins
+
+The inspector dock allows you to create custom widgets to edit properties through plugins.
+This can be beneficial when working with custom datatypes and resources, although you can
+use the feature to change the inspector widgets for built-in types. You can design custom
+controls for specific properties, entire objects, and even separate controls associated
+with particular datatypes.
+For more info, see
+[docs.godotengine.org](https://docs.godotengine.org/en/stable/classes/class_editorinspectorplugin.html#class-editorinspectorplugin).
+
+The [example](https://docs.godotengine.org/en/stable/tutorials/plugins/editor/inspector_plugins.html) in the Godot docs in Rust.
+
+Before:
+
+![Before](./images/before.png)
+
+After:
+
+![After](./images/after.png)
+
+Add this dependency to Rust with the shell in the same directory as `Cargo.toml`.
+
+```bash
+cargo add rand
+```
+
+Add file `addon.rs` and import it in `lib.rs`:
+
+```rust
+// file: lib.rs
+mod addon;
+```
+
+Add the following imports at the beginning of the file
+
+```rust
+use godot::{
+    engine::{
+        Button, EditorInspectorPlugin, EditorPlugin, EditorProperty, IEditorInspectorPlugin,
+        IEditorPlugin, IEditorProperty,
+    },
+    global,
+    prelude::*,
+};
+use rand::Rng;
+```
```

Comment:
Please use flat imports. Also, `godot::engine` has been renamed to `godot::classes`.